using-mobile-extensions/mobile-core/identity/identity-event-reference.md

```diff
@@ -4,8 +4,160 @@
 
 The following events are handled by the Identity extension:
 
+### Analytics Identity Response
+
+#### Event Details
```

Comment:
Let's use the following names & order for all events:
Event name 
Include a short description & when it is generated
**Event details**
**Data payload definition**
**Event data example**